php-nginx/build-scripts/build_php56.sh

```diff
@@ -118,6 +115,10 @@ strip ${PHP56_DIR}/bin/php ${PHP56_DIR}/sbin/php-fpm
 # Defaults to PHP5.6
 ln -s ${PHP56_DIR} ${PHP_DIR}
 
+# Install shared extensions
+${PHP56_DIR}/bin/pecl install memcache
```

Comment:
The shell script becomes simpler. Also once if we're sure that the `pecl` command is working fine, it will open a door for us to install other extensions on demand.

For example, possibly we can do that in `composer.sh`, reading `ext-foobar` type of requirements, then install the apt-dev deps, then run `pecl` command.